Spoony-iOS/Spoony-iOS/Source/Feature/Home/MapFeature.swift

```diff
@@ -106,18 +111,34 @@ struct MapFeature {
                 
             case let .setShowDailySpoonPopup(show):
                 state.showDailySpoonPopup = show
-                return .none
-                
-            case .dismissDailySpoonPopup:
-                state.showDailySpoonPopup = false
+                if !show {
+                    state.drawnSpoon = nil
+                    state.spoonDrawError = nil
+                    state.isDrawingSpoon = false
+                }
                 return .none
                 
             case .drawDailySpoon:
+                state.isDrawingSpoon = true
+                state.spoonDrawError = nil
+                
                 return .run { send in
-                    try await Task.sleep(nanoseconds: 1_000_000_000)
-                    await send(.fetchSpoonCount)
+                    let result = await TaskResult { try await homeService.drawDailySpoon() }
+                    await send(.spoonDrawResponse(result))
                 }
                 
+            case let .spoonDrawResponse(.success(response)):
+                state.isDrawingSpoon = false
+                state.drawnSpoon = response
+                
+                return .send(.fetchSpoonCount)
+                
+            case let .spoonDrawResponse(.failure(error)):
+                state.isDrawingSpoon = false
+                state.spoonDrawError = error.localizedDescription
+                print("스푼 뽑기 오류: \(error.localizedDescription)")
```

Comment:
```suggestion
                print("스푼 뽑기 오류: \(state.spoonDrawError)")
```
spoonDrawError 필드가 있으면, 이렇게 해도 문제 없으려나요~?

이미 state에 error.localizedDescription을 저장했기 때문에, 직접 쓰지 않아도 동일한 문자열이 출력될 것 같네요!